src/hooks/users.ts

```diff
@@ -1,150 +0,0 @@
-import { useMemo } from "react";
```

Comment:
[note] This file was moved to `src/hooks/users/index.ts` and then split into individual files under `src/hooks/users/*`